test/evergreen/run_format_configs.sh

```diff
@@ -21,20 +20,103 @@ fi
 
 success=0
 failure=0
+running=0
+parallel_jobs=8
+cleanup=0
+PID=""
+declare -a PID_LIST
+
+usage() {
+	echo "usage: $0 "
+	echo "    -j parallel  jobs to execute in parallel (defaults to 8)"
```

Comment:
```suggestion
	echo "    -j <num_of_jobs>  number of jobs to execute in parallel (defaults to 8)"
```

Reply:
Fixed in 107178c66c6a6792139b32ebc6fd44432ba6b88f